src/bin/rover.rs

```diff
@@ -3,10 +3,9 @@ use rover::*;
 use structopt::StructOpt;
 
 fn main() -> Result<()> {
-    logger::init();
-
     let cli = cli::Rover::from_args();
-    log::debug!("Command structure {:?}", cli);
+    timber::init(cli.log_level);
+    tracing::trace!(command_structure = ?cli);
```

Comment:
What is this syntax here doing? I haven't seen the `= ?cli` syntax before

Reply:
from [the docs](https://docs.rs/tracing/0.1.21/tracing/#recording-fields)

> The ? sigil is shorthand that specifies a field should be recorded using its fmt::Debug implementation

this allows us to print the command structure in a structured way instead of just printing the raw debug string like we were before